pkg/awsutils/awsutils.go

```diff
@@ -170,6 +173,8 @@ type EC2InstanceMetadataCache struct {
 	primaryENImac    string
 	availabilityZone string
 	region           string
+	accountID        string
```

Comment:
This is not used any more, removed in #1111

Reply:
not sure why it is showing as an add. Will update my branch and resubmit.

Reply:
Still came back! 😄

Reply:
Fixed it :)